server/routes/bookedtoday/unexpectedArrivals/index.ts

```diff
@@ -28,7 +33,17 @@ export default function routes(services: Services): Router {
   const searchForExistingRecordController = new SearchForExistingRecordController(services.expectedArrivalsService)
   get('', [searchForExistingRecordController.view()], [Role.PRISON_RECEPTION])
 
-  post('', [searchForExistingRecordController.submit()], [Role.PRISON_RECEPTION])
+  post(
+    '',
+    [
+      validationMiddleware(SearchForExistingRecordsValidation),
+      validationMiddleware(DateOfBirthValidation),
+      validationMiddleware(PrisonNumberValidation),
+      validationMiddleware(PncNumberValidation),
+      searchForExistingRecordController.submit(),
```

Comment:
maybe in the future it might be worth refactoring the validationMiddleware to accept an array of validators? Just a thought for now though.

Reply:
yes, maybe